graph-service/page_worker.py

```
@@ -0,0 +1,298 @@
+    return merged
+
+def process_page_object(object_name: str):
+    try:
+        response = minio_client.get_object(MINIO_BUCKET, object_name)
+        html_bytes = response.read()
+        text = html_bytes.decode("utf-8", errors="ignore")
+        text = convert_to_markdown(text)
+
+        ner = get_ner_model()
+        labels = ["person", "organization", "location", "project", "initiative", "job"]
+        labels = [l.lower() for l in labels]
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)